### DISCUSSION TITLE
Upgrade v4.4 to v6.5

### DISCUSSION THREAD

**Discussion by LinAungKyaw** [YetiForceCompany/YetiForceCRM#17473] [How to] [answered]
Dear All,
I would like to know, how to upgrade Yetiforce v4.4 to v6.5. Can I get the detail procedure for that and I am looking for the reply of it.
Thanks.

Best regards,
Lin

**Comment by paula-w** [comment, marked as answer]
Hi
you can find the update tutorial here: https://doc.yetiforce.com/administrator-guides/logs/updates/
make sure your server meets all the requirements: https://doc.yetiforce.com/introduction/requirements/
